app/components/add-new-goal/add-new-goal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import Avatars from './new-goal-avatars.jsx';
+
+const addGoal = () => {
+
+  return (
+    <div className="addNewGoal">
+      <div className="newGoal_inputContainer-outer">
+        <label
+          htmlFor="newGoalInput"
+          className="newGoal_label"
+        >Set a goal and choose a plant to grow with it</label>
+        <div className="newGoal_inputContainer-inner">
+          <textarea
```

Comment:
This is a https://facebook.github.io/react/docs/uncontrolled-components.html

But when using redux you want a https://facebook.github.io/react/docs/forms.html#controlled-components.

This is a pain and massive time drain, there may be some libraries that help you.

You do not need to implement this now, but as soon as you start using this textarea's value, I expect you to move its state out of the dom and into your redux state